pkg/config/config.go

```diff
@@ -159,16 +162,41 @@ func ConfigureLogging() {
 	zerolog.DefaultContextLogger = &log.Logger
 }
 
+// WrapMiddleware wraps `func(http.Handler) http.Handler` into `echo.MiddlewareFunc`
+func WrapMiddlewareWithSkipper(m func(http.Handler) http.Handler, skip middleware.Skipper) echo.MiddlewareFunc {
+	return func(next echo.HandlerFunc) echo.HandlerFunc {
+		return func(c echo.Context) (err error) {
+			if skip != nil && skip(c) {
+				err = next(c)
+			}
+			m(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+				c.SetRequest(r)
+				c.SetResponse(echo.NewResponse(w, c.Echo()))
+				err = next(c)
+			})).ServeHTTP(c.Response(), c.Request())
+			return
+		}
+	}
+}
+
+func SkipLiveness(c echo.Context) bool {
+	return c.Request().URL.Path == "/ping"
```

Comment:
we also need to support  /api/content_sources/v1.0/ping & /api/content_sources/v1/ping

maybe anything that ends in "/ping" ?

Reply:
also, when i run:

curl -v localhost:8000/ping
the body of the response is:

```
{"message":"pong"}
Bad Request: missing x-rh-identity header
```